packages/atomic/cypress/e2e/generated-answer.cypress.ts

```diff
@@ -9,12 +10,46 @@ import {
 import * as GeneratedAnswerAssertions from './generated-answer-assertions';
 import {GeneratedAnswerSelectors} from './generated-answer-selectors';
 
+const rephraseOptions = [
+  {label: 'Bullet', value: 'bullet'},
```

Comment:
I can definitely do it but is that really useful in a test?

Reply:
It would just mean that if some of the `GeneratedAnswerStyle` values changed in Headless, then the test would fail to transpile, whereas as it currently is the test would transpile, but fail to run.

So, indeed, not a big deal

Reply:
did it tho 😏